guides/V1_ActionsEvents.md

```diff
@@ -0,0 +1,220 @@
+2. User actions which can be handled by an app providing a direct user experience where the app developer codes client specific UI. Here your App will be invoked directly from the Watson Work Services client (such as Workspace) and it can provide a user experience to guide a user through the fulfillment of that action. Example: App Configuration
+
+### Implicit Action Handler
+
+These type of user actions are expected to be handled without the need for apps to provide UI code. The app is notified via a webhook and typically provides a declaration of a user experience for the Work Services client to interpret and present to the user. Examples are:
+
+1. /commands
+2. actions identified for cognitively detected intents from users utterances in a conversation
+3. clicking on buttons associated with actions presented via declarative UI
+
+
+**/Command flow**
+
+
+![Command Flow](../images/CommandFlow.png)
+
+
+**Action Selected flow**
+
+
+![Action Selected Flow](../images/actionSelectedFlow.png) 
+
+### Explicit Action Handler
+
+Directly handling a user action allows your App to deliver a user experience via a Watson Work Services client. You will need to register a call back URL that the Work Services client will present typically in a browser/webview context such as a window, iframe, etc. The Work Services client implementation chooses what user gestures and where in the user experience these user actions take place by adhering to a known list of supported user actions by Watson Work Services. The app's handler will get a token code which it will use to obtain context corresponding to the type of action the user took.
+
+Here’s the flow for client action handlers
+
+![Client Action Handler Flow](../images/ClientActionHandlerFlow.png)
+
+## Explicit Action Handler available actions
+
+There are currently two actions available, `space-app-config-requested` and `team-app-config-requested`. Other user actions will soon become available such as clicking on buttons to share resources, etc...
+
+In Summary: this pattern is utilized by Watson Work Services to allow clients adhering to its programming model to be extensible in their user experience by third party applications.
+
+| Action                        | Description       |
+| ----------------------------- |:------------------|
+| `space-app-config-requested`  | This registers an action handler to allow your app to be configured for a user in a space. Note that this was previously called the Configuration URL |
+| `team-app-config-requested`   | This registers an action handler to allow your app to be configured for a team, so that a team administrator can set the appropriate configuration for your app for their entire team|
+
+
+## Adding a client action handler callback
+
+To add a Client Action Handler for your App, you need to;
+
+1. Go to Your Apps and choose the App.
```

Comment:
We may want to have the text followed by a screen.  It is confusing now as step 1 does not have a screen and users will think this step corresponds to the picture below when that is not the case.